src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/OptionList.js

```diff
@@ -45,6 +45,7 @@ export default class OptionList extends React.PureComponent<Props> {
 
         return (
             <div>
+                <Backdrop open={true} local={true} onClick={this.handleBackdropClick} visible={false} />
```

Comment:
It is `local` because the `OptionList` is inside the `Toolbar` and not a `Popover` kind of component. We need the `local` param to achieve the needed behavior.